include/zephyr/drivers/cellular.h

```diff
@@ -52,6 +52,37 @@ struct cellular_network {
 	uint16_t size;
 };
 
+enum cellular_signal_type {
+	CELLULAR_SIGNAL_RSSI,
+	CELLULAR_SIGNAL_QUALITY,
+	CELLULAR_SIGNAL_PERCENTAGE,
+};
+
+enum cellular_signal_quality {
+	CELLULAR_MODEM_SIGNAL_EXCELLENT,
+	CELLULAR_MODEM_SIGNAL_GOOD,
+	CELLULAR_MODEM_SIGNAL_BAD,
+};
+
+typedef int16_t cellular_signal_rssi;
+
+typedef uint8_t cellular_signal_percentage;
```

Comment:
I would rather we limit ourselves to integers for now, which RSSI, RSRQ and RSRP can be represented as, if we have to use a more exotic value, we should use a union similar to  the `uart_event` data member

Reply:
ok